lnrpc/rpc.pb.go

```diff
@@ -1775,6 +1779,8 @@ type OpenChannelRequest struct {
 	Private bool `protobuf:"varint,8,opt,name=private" json:"private,omitempty"`
 	// / The minimum value in millisatoshi we will require for incoming HTLCs on the channel.
 	MinHtlcMsat int64 `protobuf:"varint,9,opt,name=min_htlc_msat" json:"min_htlc_msat,omitempty"`
+	// / The delay we require on the remote's commitment transaction. If this is not set, it will be scaled automatically with the channel size.
+	RemoteCsvDelay uint32 `protobuf:"varint,10,opt,name=remote_csv_delay" json:"remote_csv_delay,omitempty"`
```

Comment:
Any particular reason why this is typed as an unsigned int, while `MinHtlcMsat` (which sounds like it will always be a positive integer since it represents an amount in satoshis) is `int64`?

Reply:
Good question! It could certainly have been unsigned, but it was made an `Int64` because it is later converted to `lnwire.Millisatoshi` which is interpreted as `Int64`.